src/Interfaces/ClusterizationResultInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Kmeans\Interfaces;
+
+interface ClusterizationResultInterface extends \Serializable
```

Comment:
https://github.com/bdelespierre/php-kmeans/issues/31

We decided to add getTotalVariance() .

Reply:
Yes, in another PR +1